charts/tfy-k8s-azure-aks-inframold/artifacts-manifest.json

```diff
@@ -0,0 +1,348 @@
+        }
+    },
+    {
+        "type": "helm",
+        "details": {
+            "chart": "tfy-istio-ingress",
+            "repoURL": "https://truefoundry.github.io/infra-charts/",
+            "targetRevision": "0.1.3"
+        }
+    },
+    {
+        "type": "image",
+        "details": {
+            "registryURL": "quay.io/argoproj/argo-rollouts:v1.6.6"
+        }
+    },
+    {
+        "type": "image",
+        "details": {
+            "registryURL": "quay.io/argoproj/workflow-controller:v3.5.5"
+        }
+    },
+    {
+        "type": "image",
+        "details": {
+            "registryURL": "quay.io/argoproj/argocli:v3.5.5"
+        }
+    },
+    {
+        "type": "image",
+        "details": {
+            "registryURL": "docker.io/grafana/grafana:10.0.3"
+        }
+    },
+    {
+        "type": "image",
+        "details": {
+            "registryURL": "docker.io/library/busybox:1.31.1"
+        }
+    },
+    {
+        "type": "image",
+        "details": {
+            "registryURL": "docker.io/bats/bats:v1.4.1"
+        }
+    },
+    {
+        "type": "image",
+        "details": {
+            "registryURL": "gcr.io/istio-release/pilot:1.21.1-distroless"
+        }
+    },
+    {
+        "type": "image",
+        "details": {
+            "registryURL": "ghcr.io/kedacore/keda:2.13.1"
+        }
+    },
+    {
+        "type": "image",
+        "details": {
+            "registryURL": "ghcr.io/kedacore/keda-metrics-apiserver:2.13.1"
+        }
+    },
+    {
+        "type": "image",
+        "details": {
+            "registryURL": "ghcr.io/kedacore/keda-admission-webhooks:2.13.1"
+        }
+    },
+    {
+        "type": "image",
+        "details": {
+            "registryURL": "gcr.io/kubecost1/cost-model:prod-2.1.0"
+        }
+    },
+    {
+        "type": "image",
+        "details": {
+            "registryURL": "gcr.io/kubecost1/frontend:prod-2.1.0"
+        }
+    },
+    {
+        "type": "image",
+        "details": {
+            "registryURL": "alpine/k8s:1.26.9"
+        }
+    },
+    {
+        "type": "image",
+        "details": {
+            "registryURL": "docker.io/grafana/promtail:2.8.3"
+        }
+    },
+    {
+        "type": "image",
+        "details": {
+            "registryURL": "docker.io/grafana/loki:2.8.4"
+        }
+    },
+    {
+        "type": "image",
+        "details": {
+            "registryURL": "public.ecr.aws/truefoundrycloud/tfy-notebook-controller:6df7e07ca442a53583ce3bcb507f249b15818152"
+        }
+    },
+    {
+        "type": "image",
+        "details": {
+            "registryURL": "quay.io/prometheus/node-exporter:v1.7.0"
+        }
+    },
+    {
+        "type": "image",
+        "details": {
+            "registryURL": "registry.k8s.io/kube-state-metrics/kube-state-metrics:v2.10.1"
+        }
+    },
+    {
+        "type": "image",
+        "details": {
+            "registryURL": "quay.io/prometheus-operator/prometheus-operator:v0.70.0"
```

Comment:
For prometheus lets also add the images needed by the custom prometheus resources like alert manager etc. This will necessitate some change in the code